var/spack/repos/builtin/packages/openkim-models/package.py

```diff
@@ -19,14 +19,18 @@ class OpenkimModels(CMakePackage):
        compatible with the kim-api package.
     """
     homepage = "https://openkim.org/"
-    url      = "https://s3.openkim.org/archives/collection/openkim-models-2019-07-25.txz"
+    url      = "https://s3.openkim.org/archives/collection/openkim-models-2021-01-28.txz"
```

Comment:
FWIW. This change is technically not required as Spack will perform a substitution.